pkg/roachprod/vm/aws/support.go

```diff
@@ -64,7 +64,7 @@ if [ "${#disks[@]}" -eq "0" ]; then
   echo "No disks mounted, creating ${mountpoint}"
   mkdir -p ${mountpoint}
   chmod 777 ${mountpoint}
-elif [ "${#disks[@]}" -eq "1" ] || [ -n "use_multiple_disks" ]; then
+elif [ "${#disks[@]}" -eq "1" ] || [ -n "$use_multiple_disks" ]; then
```

Comment:
cc: @nvanbenschoten @miretskiy - looks like this was [picked up in the original PR](https://github.com/cockroachdb/cockroach/pull/54986/files#r602445704).

I believe this has probably been broken since that  the original commit was merged (March), and we would have been creating non-RAID 0 clusters in cases where we declared that we wanted multiple SSDs attached with a single store (i.e. just `/mnt/data1`). Instead we've been creating multiple data dirs in a non-RAID configuration.

Reply:
Haha.. Nice one; goes to show just how under tested this stuff was.